ledger/accountdb.go

```diff
@@ -1971,12 +1971,14 @@ func (qs *accountsDbQueries) lookupResources(addr basics.Address, aidx basics.Cr
 				data.addrid = rowid.Int64
 				return protocol.Decode(buf, &data.data)
 			}
+			data.data = makeResourcesData(0)
 			// we don't have that account, just return the database round.
 			return nil
 		}
 
 		// this should never happen; it indicates that we don't have a current round in the acctrounds table.
 		if err == sql.ErrNoRows {
+			data.data = makeResourcesData(0)
```

Comment:
this does not look needed as we return an error and something empty

Reply:
agree